### PR TITLE
fix(swagger): fix errors in swagger

### PR DESCRIPTION
**Pull request by Darpholgshon**

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -250,328 +234,328 @@ paths:
...
+                      "organisation": {
+                        "code": "BEL",
+                        "description": "Belgium"
+                      }
+                    }
+                  }
+                ],
+                "link": [{
+                  "rel": "self",
+                  "href": "https://olympics.api.press.net/v3/games/OWG2018/unit/SBDMBA----------------------------"
```
Comment (danpetitt) [inline swagger.yaml]: Are these URLs correct? Shouldnt they be `pamedia.io` domains?

File: swagger.yaml [hunk 2]
```diff
@@ -1543,8 +1529,30 @@ parameters:
...
+  start:
+    name: start
+    in: query
+    required: false
+    type: string
+    format: 'date-time'
+    x-example: 2014-04-17
+    description: >-
+      Limit the entities to ones within the specified range.
+      The 'end' parameter is typically used in combination with the 'start' param to form a date range.
```
Comment (danpetitt) [inline swagger.yaml:1541]: Are the dates inclusive so end would include all the events on that date if it has no 'time' included; or if it has a time, it includes events that fall on that time?

File: swagger.yaml (hunk 1 above)
Comment (Darpholgshon) [reply swagger.yaml]: Good spot

File: swagger.yaml (hunk 2 above)
Comment (Darpholgshon) [reply swagger.yaml:1541]: Can't remember. Probably plugged straight into a between clause so >= start and < end

**Review by danpetitt — APPROVED** [review]

**Pull request merged**